CMSgov/mint-app
# [NOREF] Default FLYWAY_CLEAN_DISABLED to false

# NOREF

## Changes and Description

- Default `FLYWAY_CLEAN_DISABLED` to `"false"`

This was needed because [Flyway v9](https://flywaydb.org/documentation/learnmore/releaseNotes#9.0.0) introduced a change:

> Change default of cleanDisabled to true.

which caused a [recent deploy](https://github.com/CMSgov/mint-app/actions/runs/3760628705/jobs/6391758538) to fail due to Flyway erroring out

![image](https://user-images.githubusercontent.com/15203744/209221612-ba8cd707-b18c-48df-84be-919071d3f07f.png)


## How to test this change

The instructions in https://github.com/CMSgov/mint-app/pull/374 should be the same for testing, except the fact that we overrode `FLYWAY_CLEAN_DISABLED` in the `docker run` command is what made this not get caught.

Modify the one-liner in that PR to be 

```bash
FLYWAY_USER=postgres FLYWAY_PASSWORD=mysecretpassword FLYWAY_URL=jdbc:postgresql://host.docker.internal:5437/postgres docker run --env FLYWAY_USER --env FLYWAY_PASSWORD --env FLYWAY_URL $(docker build -q -f Dockerfile.db_clean .)
```

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
